platform/ui/tailwind.config.js

```diff
@@ -14,32 +14,24 @@ module.exports = {
       black: '#000',
       white: '#fff',
 
-      primary: {
-        light: '#5acde7',
-        main: '#0142b4',
-        dark: '#03123f',
-        contrastText: '#fff',
+      /** CUSTOM THEME COLORS */
+      custom: {
+        dark: '#1f1f27',
+        aquaBright: '#5acce6',
+        navyDark: '#090c29',
+        navy: '#041c4a',
+        violetDark: '#2b166b',
+        violetPale: '#3a3f99',
+        blue: '#0944b3',
+        blueBright: '#348cfd',
+        aquaPale: '#7bb2ce',
+        white: '#ffffff',
+        grayBright: '#e1e1e1',
+        grayLight: '#a19fad',
+        gray: '#726f7e',
```

Comment:
Naming is tricky. Not sure I want to have to remember _every_ single one of these. Better to use semantic meaning than the color's name as it's easier to recall. Also, I can change the `primary` color for the app, and all classes are correct. Changing "aquaBright" requires a find and replace all.

We should work with Dan to get the correct semantic names for these, and then make sure they're used consistently?